deployment/stack.py

```diff
@@ -44,12 +46,22 @@ class ConfigError(Exception):
     pass
 
 
-def main():
+def main(argv=None):
+    if argv is None:
+        argv = sys.argv
```

Comment:
Guido's main

http://www.artima.com/weblogs/viewpost.jsp?thread=4829

it has the benefit that you import `stack.py` from an interactive interpreter and then call `main()` by hand with a specific set of arguments.
Admittedly not that useful in this case, but I have used it elsewhere.

So it's habit more than anything else.